internal/terraform/terraform_test.go

```diff
@@ -0,0 +1,232 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"github.com/hazelops/ize/internal/config"
+	"github.com/hazelops/ize/pkg/terminal"
+	"io"
+	"os"
+	"testing"
+)
+
+func TestInstall(t *testing.T) {
+	type args struct {
+		tfversion string
+		mirrorURL string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{name: "success", args: args{
+			tfversion: "0.9.0",
+			mirrorURL: defaultMirror,
+		}, wantErr: false},
+		{name: "success exist", args: args{
```

Comment:
I would switch to a higher version of terraform, >1, as 0.9.0 might be deprecated at some point.